modules/k3s/variables.tf

```diff
@@ -14,7 +14,9 @@ variable "instance_class" {}
 variable "ec2_instance_class" {}
 variable "db_group_name" {}
 variable "username" {}
-variable "password" {}
+variable "password" {
+  default = "Pa$$w0rd"
```

Comment:
Is default required? Can we leave it blank?

Reply:
+1

Reply:
fixed. removed this change.